testdata/scripts/first.txt

```diff
@@ -0,0 +1,10 @@
+exec devnet &
+
+exec sleep 20
+
+env TOKEN=`cat ~/.devLotus/token`
+env API=`cat ~/.devLotus/api`
```

Comment:
command substitutions don't work here, as this is not a full shell. Assuming these are generated and non-static, one way to fix this would be to define a command like `read-env TOKEN .devLotus/token`, using Cmds in https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript#Params and a combination of https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript#TestScript.ReadFile and https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript#TestScript.Setenv.